src/modules/shopPage/components/FilterMenu/FilterFullMenu/FilterFullMenuContainer.tsx

```diff
@@ -0,0 +1,285 @@
+    Width: {
+      inputComponent: FilterFullMenuInput,
+      dataComponent: FilterFullMenuWidthData,
+      ...textData.Width,
+    },
+    Profile: {
+      inputComponent: FilterFullMenuInput,
+      dataComponent: FilterFullMenuProfileData,
+      ...textData.Profile,
+    },
+    Diametr: {
+      inputComponent: FilterFullMenuInput,
+      dataComponent: FilterFullMenuDiametrData,
+      ...textData.Diametr,
+    },
+    Price: {
+      dataComponent: FilterFullMenuPriceData,
+      ...textData.Price,
+    },
+    Season: {
+      inputComponent: FilterFullMenuInput,
+      dataComponent: FilterFullMenuSeasonData,
+      ...textData.Season,
+    },
+    Brand: {
+      inputComponent: FilterFullMenuInput,
+      dataComponent: FilterFullMenuBrandData,
+      ...textData.Brand,
+    },
+  };
+
+  function renderBoldText(key: keyof typeof FilterFullMenuData) {
+    if (menuData[key].parametr && menuData[key].textForParametr) {
+      return (
+        <StyledText>
+          The size is indicated as follows: <b>205/75 R16 91T</b>, where{" "}
+          <b>{menuData[key].parametr}</b>
+          {menuData[key].textForParametr}
+        </StyledText>
+      );
+    }
+    return null;
+  }
+
+  const tabsIcons = [
+    { label: <WidthIcon /> },
+    { label: <ProfileIcon /> },
+    { label: <DiametrIcon /> },
+    { label: <PriceIcon /> },
+    { label: <SeasonIcon /> },
+    { label: <BrandIcon /> },
+  ];
+
+  const menuKeys = Object.keys(menuData);
+
+  return (
+    <Box
+      sx={{
+        flexGrow: 1,
+        bgcolor: "background.paper",
+        display: "flex",
+      }}
+    >
+      <Tabs
+        orientation="vertical"
+        variant="scrollable"
+        value={value}
+        onChange={handleChange}
+        aria-label="Vertical tabs example"
+        scrollButtons={false}
+        TabIndicatorProps={{
+          style: {
+            backgroundColor: BASE_COLORS.DEFAULT_BLUE,
+          },
+        }}
+        sx={{
+          borderRight: 1,
+          borderColor: "divider",
+          "& .MuiTab-root.Mui-selected": {
+            color: BASE_COLORS.DEFAULT_BLUE,
+          },
+        }}
+      >
+        {tabsIcons.map((tab, index) => (
+          <Tab key={index} label={tab.label} {...a11yProps(index)} />
+        ))}
+      </Tabs>
+      {menuKeys.map((key, index) => {
+        const InputComponent = menuData[key].inputComponent;
+        const headerTitle = menuData[key].headerTitle;
+        const DataComponent = menuData[key].dataComponent;
+        return (
+          <TabPanel key={key} value={value} index={index}>
+            <Box display="flex">
+              <Box
+                flexBasis="385px"
+                sx={{
+                  display: "flex",
+                  flexWrap: "wrap",
+                  flexDirection: "column",
+                  padding: "45px 40px 0 40px",
+                  flexGrow: 1,
+                  maxWidth: "385px",
+                }}
```

Comment:
Optional:
you can pass a major part of the CSS styles as a component property

Reply:
Done